evm-tests/jsontests/src/state.rs

```diff
@@ -559,13 +559,31 @@ fn test_run(
 							let code = data;
 							let value = transaction.value.into();
 
-							let _reason = executor.transact_create(
+							let reason = executor.transact_create(
 								caller,
 								value,
 								code,
 								gas_limit,
 								access_list,
 							);
+							if let Some(expect_exception) = state.expect_exception.as_ref() {
+								match reason.0 {
+									ExitReason::Error(ref e) => match e {
+										ExitError::CreateContractLimit => {
+											// Check error message for EIP-3860
+											let check_result = matches!(
+												expect_exception.as_str(),
+												"TR_InitCodeLimitExceeded"
+													| "TransactionException.INITCODE_SIZE_EXCEEDED"
+											);
+											assert!(check_result, "message: {expect_exception}");
+											continue;
```

Comment:
Because 1 loop iteration is 1 test. If I'll break the loop I'll brew other tests pass. And I'm not sure that we should compare state results if inside ethereum-test contain a special field "expect_exception". Same flow in REVM (but they even don't check error message and skip state check). And yes state hash after exception is different than expected in json test with an exception.